bscript/interpreter/sigcache.go

```diff
@@ -0,0 +1,119 @@
+// Copyright (c) 2015-2016 The btcsuite developers
+// Use of this source code is governed by an ISC
+// license that can be found in the LICENSE file.
+
+package interpreter
+
+import (
+	"sync"
+
+	"github.com/libsv/go-bk/bec"
+)
+
+// sigCacheEntry represents an entry in the SigCache. Entries within the
+// SigCache are keyed according to the sigHash of the signature. In the
+// scenario of a cache-hit (according to the sigHash), an additional comparison
+// of the signature, and public key will be executed in order to ensure a complete
+// match. In the occasion that two sigHashes collide, the newer sigHash will
+// simply overwrite the existing entry.
+type sigCacheEntry struct {
```

Comment:
not 100% how i feel about this sigcache stuff but i guess it's okay for now

Reply:
I just carried it over because it was in the original codebase. I've no issue dropping it.